integration/src/test/resources/integration/homepage_to_competencypage.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Viewing the competency page
+  Scenario: Employee can click from the homepage to the competency page
+    Given as a kainos employee I want to get to the competencies per band page from the home page
+    When I click on the bands button
+    When I click on the <band> heading
```

Comment:
Instead of repeating `When` which reads a bit awkwardly, you can use `And`
```suggestion
    When I click on the bands button
    And I click on the <band> heading
```
This reads more naturally to me. What do you think?